### PR TITLE
FAQs.md

### PR DESCRIPTION
**Pull request by HarryHeights**
This is the FAQs document. Please point out where you need corrections to be made. Thanks.

### CODE REVIEW

File: FAQs.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+- No, NoteCrypt does not support markdown text styling, but the formats can be inputted when creating a note and you can preview with on a platform that supports markdown.
+
+**What's the purpose of tags?**
+
+- With tags, you can quickly access notes that carry that tag from the tag filter. For example, you tag a note with software. When you click the tag filter and select software, it brings out all the notes that have the software tag.
+
+**Can I make a manual backup of my database?** 
+
+- Yes, you can make a manual backup of a note database. The database is stored in a location on your device storage name ```NoteCrypt```. From that folder, you can see all your database file and can copy them to another location or send to your cloud storage.
```
Comment (RyuzakiKK) [inline FAQs.md]: "name" to "named"?

File: FAQs.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+- Tag filter
+- Favorite note
+- Note sorting
+- The database is securely encrypted with AES 128 bit, using cipher-block chaining (CBC), PKCS5Padding and a random IV generated at every save. For the KDF a random salt and PBKDF2WithHmacSHA1 are used.
+- Auto create a hidden backup database to prevent data loss in case the file gets corrupted while saving (the backup file is saved with the name: ."database name").
+- Note search.
+
+**What is the minimum length of characters to set a password?**
+
+- The minimum password for a database is one character, however, it's not advisable to secure a database with just a single digit or alphabet. NoteCrypt allows numeric and alphabetic password entry. For more security, create a password mixed with alphabets and numbers up to 8 characters.
```
Comment (RyuzakiKK) [inline FAQs.md]: At the end we are talking of ways to achieve more security. So instead of "up to" it should be "at least 8 characters"

**Review by RyuzakiKK — CHANGES_REQUESTED** [review]
Very good!
Just a couple of minor fixes and it is perfect.

File: FAQs.md (hunk 2 above)
Comment (HarryHeights) [reply FAQs.md]: Alright.

File: FAQs.md (hunk 1 above)
Comment (HarryHeights) [reply FAQs.md]: Yes you are right.

**Review by RyuzakiKK — APPROVED** [review]

**Pull request merged**